transfers/utils.py

```diff
@@ -39,6 +40,52 @@ def _get_ts2ps_application_status(userprofile, ts2ps):
     else:
         return 1
 
+def get_deadline_status(form_type):
+    if(form_type == TransferType.PS2TS.value):
+        if datetime.datetime.now() < DeadlineModel.objects.first().deadline_PS2TS:
+            return True
+    else:
+        if datetime.now() < DeadlineModel.objects.first().deadline_TS2PS:
+            return True
+    return False
+
+
+def get_PSD_data(self, request, *args, **kwargs):
+    response = {}
+    try:
+        DeadlineModel.objects.first().deadline_PS2TS
+
+        deadline_PS2TS = DeadlineModel.objects.first().deadline_PS2TS
+        deadline_TS2PS = DeadlineModel.objects.first().deadline_TS2PS
+        is_active_PS2TS = DeadlineModel.objects.first().is_active_PS2TS
+        is_active_TS2PS = DeadlineModel.objects.first().is_active_TS2PS
+        message = DeadlineModel.objects.first().message
+
+        response['data'] = {
+            'deadline_PS2TS': deadline_PS2TS,
+            'deadline_TS2PS': deadline_TS2PS,
+            'is_active_PS2TS': is_active_PS2TS,
+            'is_active_TS2PS': is_active_TS2PS,
+            'message': message
+        }
+        response['error'] = False
+        response['message'] = 'success'
+    except Exception as e:
+        response['data'] = {}
+        response['error'] = True
+        response['message'] = 'error'
+    return JsonResponse(response, safe=False)
```

Comment:
This function returns a response, will not come here. Move it somewhere inside `psd_views`